application/src/main/java/org/opentripplanner/updater/siri/SiriTimetableSnapshotSource.java

```diff
@@ -45,12 +42,12 @@
  * necessary to provide planning threads a consistent constant view of a graph with real-time data at
  * a specific point in time.
  */
-public class SiriTimetableSnapshotSource implements TimetableSnapshotProvider {
+public class SiriTimetableSnapshotSource {
```

Comment:
Am I correct in understanding that the [Gtfs]TimetableSnapshotSource and SiriTimetableSnapshotSource are no longer sources of timetable snapshots at all, but have become translating or mapping code which interprets messages in two different incoming formats, applying both of them to the same modification API on a single instance of TimetableSnapshotManager injected by Dagger, particularly via `snapshotManager.updateBuffer(realTimeTripUpdate)`?

Reply:
Yes, that is absolutely correct.

However, since this PR is complex enough as it is, I chose to defer any kind of renaming to a follow up where I will undoubtedly produce a lot of discussion.